### PR TITLE
⭐ Framework cmd

### PR DESCRIPTION
**Pull request by slntopp**

### CODE REVIEW

**Review by imilchev — APPROVED** [review]
the code looks good. @tas50 perhaps you want to try it out before releasing?

File: apps/cnspec/cmd/framework.go [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+	frameworkDownloadCmd.Flags().StringP("file", "f", "", "output file")
+	frameworkCmd.AddCommand(frameworkDownloadCmd)
+	// upload
+	frameworkUploadCmd.Flags().StringP("file", "f", "", "input file")
+	frameworkCmd.AddCommand(frameworkUploadCmd)
+}
+
+var frameworkCmd = &cobra.Command{
+	Use:   "framework",
+	Short: "Manage local and upstream compliance frameworks",
```
Comment (tas50) [inline apps/cnspec/cmd/framework.go]:
```suggestion
	Short: "Manage local and Mondoo Platform hosted compliance frameworks",
```

File: test/cli/testdata/cnspec.ct [hunk 2]
```diff
@@ -12,6 +12,7 @@ Usage:
 
 Available Commands:
   completion  Generate the autocompletion script for the specified shell
+  framework   Manage local and upstream compliance frameworks
```
Comment (tas50) [inline test/cli/testdata/cnspec.ct]:
```suggestion
  framework   Manage local and Mondoo Platform hosted compliance frameworks
```

**Comment by slntopp** [comment]
@tas50 updated it, is that what you expect, or should we make more elaborate error message, like "minimal required permission is X"?

**Pull request merged**